### PR TITLE
Cfy 2828 hosts ui

### PR DESCRIPTION
**Pull request by SefiGiga**

### CODE REVIEW

File: source/guide/3.2/getting-started-execute-workflow.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ cfy executions start -w <WORKFLOW_NAME> -d <DEPLOYMENT_NAME>
...
+
+Click the 'run execution' play button to start the workflow execution.
+![workflows3](/guide/images/ui/ui-workflows3.png)
+
+After the 'run execution' button is clicked, the progress spinner is displayed. The execution can be cancelled by clicking on the 'cancel execution' X button.
+ ![workflows4](/guide/images/ui/ui-workflows4.png)
+
 When executing a `Workflow` for a `Deployment` (e.g. the `install` workflow), the topology nodes show badges that reflect the workflow execution state.
 
 ## Badges
```
Comment (nir0s) [inline source/guide/3.2/getting-started-execute-workflow.md]: the badges part really shouldn't be here.. it has nothing to do with "Getting Started". Please move it to the Web UI's section.

File: source/guide/3.2/webui-hosts-page.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (nir0s) [inline source/guide/3.2/webui-hosts-page.md]: brilliant!

File: source/guide/3.2/webui-hosts-page.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+You can control which hosts are displayed by filtering by:
+
+# Blueprints
+See the definition [here]({{page.terminology_link}}#blueprint).<br/>
+![Blueprints](/guide/images/ui/ui-hosts-blueprints-selection.png)
+
+# Deployments
+See the definition [here]({{page.terminology_link}}#deployment).<br/>
+This list is affected by the selection of blueprints. <br/>
+If the blueprints selection is consisted only of undeployed blueprints, this dropdown will not be visible.<br/>
```
Comment (nir0s) [inline source/guide/3.2/webui-hosts-page.md]: is consisted -> consists

File: source/guide/3.2/webui-logs-events-page.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+You can control which events are displayed by filtering by:
+
+# Blueprints
+See the definition [here]({{page.terminology_link}}#blueprint).<br/>
+![Blueprints](/guide/images/ui/ui-logsNevents-blueprint-selection.png)
+
+# Deployments
+See the definition [here]({{page.terminology_link}}#deployment).<br/>
+This list is affected by the selection of blueprints. <br/>
+If the blueprints selection is consisted only of undeployed blueprints, this dropdown will not be visible.<br/>
```
Comment (nir0s) [inline source/guide/3.2/webui-logs-events-page.md]: is consisted -> consists

**Comment by SefiGiga** [comment]
All your comments were addressed.

**Pull request merged**